jobs/JEVS_GLOBAL_ENS_PREP

```diff
@@ -66,7 +67,7 @@ if [ $RUN = headline ]; then
 elif [ $RUN = atmos ]; then
```

Comment:
Also, this:

export COMOUT=${COMOUT:-$(compath.py -o $NET/${evs_ver})}/$STEP/${COMPONENT}

needs to be changed to this:
export COMOUT=${COMOUT:-$(compath.py -o $NET/${evs_ver}/$STEP/${COMPONENT)}

assuming I have my braces and parenthesis correct.